src/model/quotes.js

```diff
@@ -178,6 +177,34 @@ class QuotesModel {
         await this.db.getParticipant(quoteRequest.payee.partyIdInfo.fspId, LOCAL_ENUM.PAYEE_DFSP, quoteRequest.amount.currency, ENUM.Accounts.LedgerAccountType.POSITION)
       }
     }
+
+    // check if the payer is active fsp, if not send error callback to payer
+    if (payer.data.isActive === 0) {
+      throw ErrorHandler.CreateFSPIOPError(ErrorHandler.Enums.FSPIOPErrorCodes.PAYER_FSP_ID_NOT_FOUND,
+          `Payer FSP ID not found - Unsupported participant '${fspiopSource}'`, null, fspiopSource)
+    }
+    // check if the payee is active fsp, if not send error callback to payer
+    if (payee.data.isActive === 0) {
+      throw ErrorHandler.CreateFSPIOPError(ErrorHandler.Enums.FSPIOPErrorCodes.DESTINATION_FSP_ERROR,
+          `Destination FSP Error - '${fspiopDestination}' is inactive`, null, fspiopSource)
+    }
+
+    const payerAccounts = Array.isArray(payer.data.accounts) ? payer.data.accounts : []
```

Comment:
This sort of validation should be handled by `fetchParticipantInfo` for a cleaner separation of oncerns